content/rest/messages.textile

```diff
@@ -159,9 +159,19 @@ bc[go]. channel.Publish("event", "This is my payload")
 
 h4(#idempotent). Enabling idempotent publishing
 
-It is possible for a client publishing through REST to not receive an acknowledgement of receipt from Ably for numerous reasons, which can lead to the client re-publishing a message. If you wish to ensure that Ably can recognise these re-sent messages as duplicates, you should "enable idempotent publishing":/rest/usage#client-options. This attaches an identifier to each message sent from the client, enabling Ably to detect duplicates of the same message and ignore them.
+Idempotency ensures that multiple publishes of the same message cannot result in duplicates messages. "Find out more about how what idempotency is, and how we provide idempotency in our REST operations.":https://www.ably.io/concepts/idempotency
 
-If you wish to use your own IDs for idempotent publishing, you can specify it when you "publish a message":/rest/channels#publish as the "id":#id element of the "Message":#properties object. At present sending multiple messages in a single publish with client-specified "ids":#id is "not fully supported":https://support.ably.io/solution/articles/3000088357-how-can-i-publish-multiple-messages-in-a-single-rest-request-with-idempotency.
+It is possible for a client publishing through REST to not receive an acknowledgement of receipt from Ably for numerous reasons such as network failures outside of our control. This can lead to the client automatically re-publishing the message, which is a feature of our SDKs to help route around network failures. We recommend that you "enable idempotent publishing":/rest/usage#client-options in our REST SDKs if you want to avoid these retry attempts potentially resulting in duplicate messages. When enabled, the REST SDK automatically assigns a "unique ID":#id to each message ensuring subsequent retries cannot result in duplicates.
```

Comment:
Link to "which is a feature of our SDKs to help route around network failures" is gone?